### PR TITLE
updated readme to reflect recent changes in the documentation structure.

### PR DESCRIPTION
**Pull request by degville**
- updated to reflect new documentation structure
- refactored URLs that aren't part of the text (as these will need to be changed alongside the text itself)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -200,3 +173,13 @@ Below are a few helpful aliases that have been suggested:
     # for the pull request.
     # git qa-pr maas-docs 6 qa-sticky-headers
     qa-pr = "!sh -c 'git checkout develop; git pull $0 develop; git checkout -b $2; git fetch-pr $0 $1; git merge pr/$1'"
+
+[contributing]: http://maas.io/docs/contributing
+[githubmaas]: https://github.com/CanonicalLtd/maas-docs
+[githubaccount]: https://git.wiki.kernel.org/index.php/Aliases
+[configuregit]:https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup
+[authtoken]: https://help.github.com/articles/creating-an-access-token-for-command-line-use/
+[cachepassword]:ttps://help.github.com/articles/caching-your-github-password-in-git/
```
Comment (pmatulis) [inline README.md]: `<space>h`

File: README.md [hunk 2]
```diff
@@ -200,3 +173,13 @@ Below are a few helpful aliases that have been suggested:
     # for the pull request.
     # git qa-pr maas-docs 6 qa-sticky-headers
     qa-pr = "!sh -c 'git checkout develop; git pull $0 develop; git checkout -b $2; git fetch-pr $0 $1; git merge pr/$1'"
+
+[contributing]: http://maas.io/docs/contributing
+[githubmaas]: https://github.com/CanonicalLtd/maas-docs
+[githubaccount]: https://git.wiki.kernel.org/index.php/Aliases
+[configuregit]:https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup
```
Comment (pmatulis) [inline README.md]: `<space>`

File: README.md [hunk 3]
```diff
@@ -200,3 +173,13 @@ Below are a few helpful aliases that have been suggested:
...
+[authtoken]: https://help.github.com/articles/creating-an-access-token-for-command-line-use/
+[cachepassword]:ttps://help.github.com/articles/caching-your-github-password-in-git/
+[remotebranch]: ttps://github.com/webmat/git_remote_branch
```
Comment (pmatulis) [inline README.md]: `h`

File: README.md [hunk 4]
```diff
@@ -200,3 +173,13 @@ Below are a few helpful aliases that have been suggested:
     # for the pull request.
     # git qa-pr maas-docs 6 qa-sticky-headers
     qa-pr = "!sh -c 'git checkout develop; git pull $0 develop; git checkout -b $2; git fetch-pr $0 $1; git merge pr/$1'"
+
+[contributing]: http://maas.io/docs/contributing
```
Comment (pmatulis) [inline README.md]: Should we be using the internal link style?

**Comment by pmatulis** [comment]
Should we be preserving the .html extensions on external links? I'm not sure anymore.

File: README.md (hunk 4 above)
Comment (degville) [inline README.md]: I think we have to - I just tested that link with the html added and it doesn't work.

File: README.md (hunk 1 above)
Comment (degville) [inline README.md]: thanks - sorry for missing these.

File: README.md (hunk 3 above)
Comment (degville) [inline README.md]: shaky vim keyboard skills.

**Pull request merged**